src/NLog.Extensions.Logging/project.json

```diff
@@ -29,18 +28,18 @@
                 "System.Xml.XmlSerializer": "4.0.0.0"
             },
             "dependencies": {
-                "NLog": "4.0.1"
+                "NLog": "4.3.4"
             }
         },
-        "dnxcore50": {
+        "netstandard1.0": {
```

Comment:
first beta for RC2 is now live: https://www.nuget.org/packages/NLog/4.4.0-beta6

netstandard1.3! :)